src/app/components/ThemeProvider/themes/sinhala.ts

```diff
@@ -19,9 +20,9 @@ const sinhalaTheme = {
   typography: {
     script: sinhaleseScript,
     fontVariants: sinhalaFontVariants,
-    fontFaces: [NOTO_SERIF_SINHALA_BOLD, NOTO_SERIF_SINHALA_REGULAR],
+    fontFaces: [NOTO_SERIF_SINHALA_REGULAR, NOTO_SERIF_SINHALA_BOLD],
```

Comment:
order again? (not big deal, I'm just interested if it has some functional purpose)